configure.ac

```diff
@@ -512,23 +512,29 @@ EXTRA_WARNINGS=""
 
 WARNLIST="
 	all
+	extra
+	unused
```

Comment:
-Wunused combined with -Wextra provides insight into several categories of code where a value is create but never used. This can encompass return values being ignored, or variables that are assigned to but never used. Generally situations like this imply that a function that can potentially return an error is having that possible error ignored.